pkg/remote/codec/thrift/thrift.go

```diff
@@ -257,7 +257,7 @@ type MessageReader interface {
 
 // MessageReaderWithMethodWithContext read from thrift.TProtocol with method
 type MessageReaderWithMethodWithContext interface {
-	Read(ctx context.Context, method string, oprot thrift.TProtocol) error
+	Read(ctx context.Context, method string, isClient bool, dataLen int, oprot thrift.TProtocol) error
```

Comment:
no need to add `isClient` param too.

Reply:
Add a comment indicating that `dataLen` represents the length of the remaining data in the input buffer.